express-notes-api/index.js

```diff
@@ -0,0 +1,125 @@
+const express = require('express');
+const json = require('./data.json');
+const parsedNotes = json.notes;
+let parsedNextId = json.nextId;
+const app = express();
+const exJSON = express.json();
+const fs = require('fs');
+app.use(exJSON);
+const notesArray = [];
+
+app.get('/api/notes', (req, res) => {
+  for (const prop in parsedNotes) {
+    notesArray.push(parsedNotes[prop]);
+  }
+  res.status(200).json(notesArray);
+});
```

Comment:
Performing multiple `GET` requests in succession should always output the same notes.